compose-examples/lodestar/README.md

```diff
@@ -0,0 +1,55 @@
+# Prysm standalone fullstack
+
+## Requirements
+* Get to know [Lodestar](https://chainsafe.github.io/lodestar/) a bit
+* Server with 4 (v)cpus & 8 gb memory & 100 gb storage
+
+## Services
+* geth (beacon connects to it to see deposits for validators)
+* beacon
+
+**All services are enabled by default.**
+
+## Validator accounts with launchpad
```

Comment:
Please remove until staking with loadstar is working

Reply:
okey, got it and just removed